src/routes/console/project-[project]/auth/security/+page.svelte

```diff
@@ -198,4 +221,28 @@
             </Button>
         </svelte:fragment>
     </CardGrid>
+
+    <CardGrid>
+        <Heading tag="h2" size="6">Session Limit</Heading>
+        <p>
+            Maximum number of active sessions allowed per user per project.
```

Comment:
```suggestion
            Maximum number of active sessions allowed per user.
```

The last part is redundant when inside a project scope.